### PR TITLE
TASK: Use type casting instead of intval/floatval

### PR DESCRIPTION
**Pull request by simonschaufi**
Use type casting instead of slower function calls.

### CODE REVIEW

**Comment by kdambekalns** [comment]
**Checklist**

- [x] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [x] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)

**Review by kdambekalns — APPROVED** [review]
Fine with me

**Comment by kdambekalns** [comment]
BTW: Flow 5.3 is no longer supported, this should go into 6.3 directly. See https://www.neos.io/features/release-process.html

**Comment by simonschaufi** [comment]
Then the readme is outdated. It still says 5.3. Could you please adjust that?

**Comment by kdambekalns** [comment]
> Then the readme is outdated. It still says 5.3. Could you please adjust that?

Done in 6.3 (https://github.com/neos/flow-development-collection/blob/6.3/Readme.rst), will be upmerged eventually…

File: Neos.Eel/Classes/Helper/MathHelper.php [hunk 1]
```diff
@@ -398,7 +398,7 @@ public function randomInt($min, $max)
      * The precision defines the number of digits after the decimal point.
      * Negative values are also supported (-1 rounds to full 10ths).
      *
-     * @param float $subject The value to round
+     * @param mixed $subject The value to round
```
Comment (kdambekalns) [inline Neos.Eel/Classes/Helper/MathHelper.php:401]: This makes Psalm happy and is technically correct, as the method handles any type.

**Review by albe — APPROVED** [review]

**Pull request merged**